src/main/java/stsjorbsmod/JorbsMod.java

```diff
@@ -59,7 +61,7 @@
     private static final String AUTHOR = "https://mod.jorbs.tv/credits";
     private static final String DESCRIPTION = "The Wanderer, CULL, and Explorer. Inspired by Jorbs and co-developed by the community.";
 
-    public static class JorbsCardTags {
+        public static class JorbsCardTags {
```

Comment:
Was this intended? This formatting isn't consistent with rest of project

Reply:
Oops probably just pressed Enter on accident.